index.html

```diff
@@ -543,7 +543,7 @@ <h4 id="return-values">Return Value</h4>
     <p>A request initiated by sendBeacon is subject to following properties:</p>
     <ul>
       <li>If the payload is one of <code>application/x-www-form-urlencoded</code>, <code>multipart/form-data</code>, or <code>text/plain</code>, then the request is a simple request that does not require an additional CORS-preflight; same as scripted form-post or XHR/fetch.</li>
```

Comment:
I think the argument here is that if you have a payload of that type, you can use XHR/fetch to send it without an additional CORS-preflight.. Unless I'm missing something and that's not the case?